cns/service/main.go

```diff
@@ -871,6 +870,32 @@ func main() {
 		}
 	}
 
+	// AzureHost channelmode indicates Nodesubnet. IPs are to be fetched from NMagent.
+	if config.ChannelMode == cns.AzureHost {
+		if !cnsconfig.ManageEndpointState {
+			logger.Errorf("ManageEndpointState must be set to true for AzureHost mode")
+			return
+		}
+
+		// If cns manageendpointstate is true, then cns maintains its own state and reconciles from it.
```

Comment:
Needs to start lowercase letters

Reply:
@behzad-mir this code is actually refactored, this comment was simply moved here as part of the refactoring. There is this and a few other comments that are already in our code that seem to start with an upper case letter. Just trying to be sure before editing pre-existing comments. Do we have some kind of a style guide about lower case in comments inside functions? I tried finding guidelines online, but couldn't find any for this case.

Reply:
@behzad-mir I'm not aware of any internal style restricting casing on comments within functions. Generally, yes, they're lowercase, but I'm also fine with anything that helps get the point across. We can use all of UTF-8 here.

The only style guide we _should_ follow is https://go.dev/doc/comment . I also very much appreciate when people leave doc comments, but we have so many undocumented ones that I understand when we don't.

Reply:
I've edited the comments that I added,  left the comments that were already present untouched.